tv_apps/browser/index.html

```diff
@@ -1,364 +1,742 @@
-    <!-- CSS reset -->
-    <link rel="stylesheet" href="style/reset.css">
-    <!-- Shared CSS-->
-    <link rel="stylesheet" href="shared/style/headers.css">
-    <link rel="stylesheet" href="shared/style/input_areas.css">
-    <link rel="stylesheet" href="shared/style/status.css">
-    <link rel="stylesheet" href="shared/style/confirm.css">
-    <link rel="stylesheet" href="shared/style/action_menu.css">
-    <link rel="stylesheet" href="style/modal_dialog/modal_dialog.css">
-    <link rel="stylesheet" href="style/modal_dialog/prompt.css">
-    <link rel="stylesheet" href="style/themes/default/core.css">
+    <title>web browser</title>
 
     <!-- Specific style -->
-    <link rel="stylesheet" href="style/layout.css" type="text/css">
-    <link rel="stylesheet" href="style/toolbar.css" type="text/css">
-    <link rel="stylesheet" href="style/awesomescreen.css" type="text/css">
+    <link rel="stylesheet" type="text/css" href="style/index.css">
+    <link rel="stylesheet" type="text/css" href="style/awesomescreen.css">
+    <link rel="stylesheet" type="text/css" href="style/authentication_dialog.css">
+    <link rel="stylesheet" type="text/css" href="style/browser_dialog.css">
+    <link rel="stylesheet" type="text/css" href="style/toolbar.css">
+    <link rel="stylesheet" type="text/css" href="style/tooltip.css">
+    <link rel="stylesheet" type="text/css" href="style/search_result.css">
+    <link rel="stylesheet" type="text/css" href="style/settings.css">
 
     <!-- Localization -->
+<!--     <link rel="resource" type="application/l10n" href="shared/locales/branding.ini"> -->
+
+    <script defer="" src="/shared/js/l10n.js"></script>
+
+    <!-- L10n properties -->
     <meta name="defaultLanguage" content="en-US">
-    <meta name="availableLanguages" content="en-US">
-    <link rel="localization" href="/shared/locales/branding/branding.{locale}.properties">
-    <link rel="localization" href="locales/browser.{locale}.properties">
+    <meta name="availableLanguages" content="de-DE:201506220909, es-ES:201506220909, nl-NL:201506220909, en-IE:201506220909, en-GB:201506220909, no-NO:201506220909, fi-FI:201506220909, fr-FR:201506220909, da-DK:201506220909, tr-TR:201506220909, it-IT:201506220909, sv-SE:201506220909, pt-PT:201506220909, el-GR:201506220909, hu-HU:201506220909, hr-HR:201506220909, cs-CZ:201506220909, ru-RU:201506220909, bg-BG:201506220909, ro-RO:201506220909, sk-SK:201506220909, sl-SI:201506220909, pl-PL:201506220909, sr-YU:201506220909, et-EE:201506220909, lv-LV:201506220909, lt-LT:201506220909, en-AU:201506220909, en-HK:201506220909, zh-CN:201506220909, ar:201506220909, fa:201506220909, th:201506220909, vi:201506220909, id:201506220909, zh-HK:201506220909, en-BR:201506220909, pt-BR:201506220909, es-BR:201506220909, hi:201506220909, zh-TW:201506220909, en-TW:201506220909, ja-JP:201506220909, en-US:201506220909, es-US:201506220909, fr-CA:201506220909, ru:201506220909, he:201506220909, my:201506220909, mk-MK:201506220909">
```

Comment:
Seems we only need to keep `en-US.properties` and `content="en-US"`.
(https://bugzilla.mozilla.org/show_bug.cgi?id=1099321)